README.md

```diff
@@ -32,18 +32,19 @@ USERNAME
 PASSWORD
 ```
 
-7. Run `docker-compose build`
+7. Run `sudo docker-compose  -f docker-compose.yaml -f docker-compose.prod.yaml build`
```

Comment:
sudo is not required on a properly configured system

Reply:
ok